cli/cmd/root.go

```diff
@@ -111,7 +113,13 @@ func (s *options) appendIfNotEmpty(newOption string) {
 	}
 }
 
-func runVersionCheck(vChecker *version.VersionChecker) {
+func runVersionCheck(vChecker *version.VersionChecker, flags []string) {
+	// Server version won't be available during `install`
+	// because the Server is not installed yet
+	if isInstallSubCommand(flags) {
```

Comment:
Doing this in `runVersionCheck` instead of `PersistentRun` because it is easier to test `runVersionCheck` (we already have code for doing this [here](https://github.com/keptn/keptn/blob/d1555f9f6882742d3d26dd4157358e6769548fb7/cli/cmd/root_test.go#L148))